.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,10 @@
+<!--
+Please if you have a question rather than reporting the issue, please post it on Stackoverflow
+or ask it on A-Frame Slack #questions channel
+https://stackoverflow.com/questions/ask/?tags=aframe
+https://aframevr.slack.com/messages/questions/
```

Comment:
Second link only works if you're already a member of the Slack. Is it still worth including, or will that confuse new people?